### PR TITLE
Add sensor entities to the LIFX docs

### PR DESCRIPTION
**Pull request by Djelibeybi**
## Proposed change

Adds a section that lists the available sensor entities for the LIFX integration, i.e. the HEV cycle status sensor, infrared brightness sensor and proposed wifi signal strength (RSSI) sensor. 

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/80993
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/lifx.markdown [hunk 1]
```diff
@@ -57,6 +57,16 @@ To determine whether or not a HEV cycle is currently running, Home Assistant exp
...
 
+## Sensors
+
+The following table lists the sensors that are available for LIFX bulbs:
+
+| Sensor | LIFX model(s) | Enabled by default? |
+| ------ | ---------- | ------------------- |
+| Clean (HEV) cycle state | LIFX Clean | Yes
+| Infrared brightness | LIFX Nightvision | Yes
+| WiFi signal strength (RSSI)   | All        | No
```
Comment (frenck) [inline source/_integrations/lifx.markdown]:
```suggestion
| Wi-Fi signal strength (RSSI)   | All        | No
```

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Review by bdraco — CHANGES_REQUESTED** [review]
Please resolve conflicts

**Review by bdraco — CHANGES_REQUESTED** [review]
It looks like there are unrelated changes in this PR

**Comment by Djelibeybi** [comment]
It's the same content, I just rearranged things a bit to make the theme section a little easier to read.

File: source/_integrations/lifx.markdown [hunk 2]
```diff
@@ -66,15 +66,27 @@ To determine whether or not a HEV cycle is currently running, Home Assistant exp
...
+| Clean cycle | Indicates whether a HEV cycle is currently active on a LIFX Clean bulb |
+| Infrared brightness | Controls the infrared brightness amount on a LIFX Nightvision bulb |
+| RSSI | Indicates the current WiFi signal strength on any LIFX bulb (disabled by default) |
 
-The following themes are available: `autumn`, `blissful`, `cheerful`, `dream`, `energizing`, `epic`, `exciting`, `focusing`, `halloween`, `hanukkah`, `holly`, `independence_day`, `intense`, `mellow`, `peaceful`, `powerful`, `relaxing`, `santa`, `serene`, `soothing`, `sports`, `spring`, `tranquil`, `warming`.
+Note that these sensors are only updated every 30 seconds and may not reflect the current state if changes are made externally to Home Assistant.
+
+## Themes
+
+Home Assistant provides a collection of predefined themes for LIFX multizone lights, each of which is designed to mimic the theme of the same name from the LIFX smartphone app.
```
Comment (bdraco) [inline source/_integrations/lifx.markdown]:
```suggestion
Home Assistant provides a collection of predefined themes for LIFX multizone lights, each designed to mimic the theme of the same name from the LIFX smartphone app.
```

**Review by bdraco — APPROVED** [review]

**Pull request merged**